modules/install-central-operator.adoc

```diff
@@ -19,13 +19,14 @@ When you install {product-title} for the first time, you must first install the
 .Procedure
 . On the {ocp} web console, navigate to the *Operators* -> *Installed Operators* page.
 . Select the {product-title} Operator from the list of installed Operators.
-. By default, {ocp} lists the project as `openshift-operator`. Select *Project: openshift-operator* -> *Create project*.
+. {ocp} lists the project as `rhacs-operator` if you have installed the Operator in the recommended namespace. Select *Project: rhacs-operator* -> *Create project*.
```

Comment:
This sentence was a little hard to read especially without haing a preview. Perhaps starting it with the "if you have installed....  would make it smoother? Or, consider adding a comma before the "if".